Dockerfile

```diff
@@ -1,5 +1,5 @@
 #--------- Generic stuff all our Dockerfiles should start with so we get caching ------------
-FROM tomcat:8.0-jre8
+FROM tomcat:8-jre8
```

Comment:
Can you try the following
```
ARG image_version=8.0-jre8
FROM tomcat:${image_version}
```
@nk027 This will be better since the only change would be the build arguments